openapi.yml

```diff
@@ -239,7 +239,7 @@ paths:
                   exp:
                     type: string
                     description: When the token will expire
-                    format: date-time
```

Comment:
The format that is used does not conform with date-time. This is now strictly enforced.